streamlit_web.py

```diff
@@ -652,5 +657,38 @@ def interactive():
         st.write(summaries)
 
 
+def entities():
+    """Page to display entity analysis"""
+    st.header("Entity analysis inspects the given text for known entities \
+    and returns information about those entities. It is a way to extract \
+    information that seeks to locate and classify named entities in text \
+    into pre-defined categories such as the names of persons, organizations, \
+    locations, expressions of times, quantities, monetary values, and percentages.")
```

Comment:
This long text should not be a header. Instead, you can use `st.write()` or other supported streamlit functions to write it  as paragraph without making the font large and bold